src/pages/tBTC/Bridge/components/DepositDetailsStep.tsx

```diff
@@ -53,108 +33,66 @@ type CommonStepProps = Pick<BridgeProcessStepProps, "onComplete"> & {
 }
 
 export const Step1: FC<
-  { confirmations?: number; requiredConfirmations?: number } & Pick<
-    BridgeProcessStepProps,
-    "txHash" | "onComplete"
-  >
-> = ({ confirmations, requiredConfirmations, txHash, onComplete }) => {
-  const subtitle = `Your Bitcoin deposit transaction requires ${requiredConfirmations} confirmation${
-    requiredConfirmations !== undefined && requiredConfirmations > 1 ? "s" : ""
-  } on the Bitcoin Network before initiating the minting process.`
-
+  { confirmations?: number; requiredConfirmations?: number } & CommonStepProps
+> = ({ confirmations, requiredConfirmations, onComplete }) => {
+  const hasConfirmations = !!confirmations && !!requiredConfirmations
   return (
     <BridgeProcessStep
-      title="Waiting for the Bitcoin Network Confirmations..."
-      chain="bitcoin"
-      txHash={txHash}
-      icon={<BTCConfirmationsIcon />}
-      progressBarColor="brand.500"
-      progressBarValue={confirmations}
-      progressBarMaxValue={requiredConfirmations}
+      loaderLabel="Minting"
+      loaderProgress={
+        0.25 * (hasConfirmations ? confirmations / requiredConfirmations : 1)
+      }
+      headingPrimary="Waiting for the Bitcoin Network Confirmations..."
+      headingSecondary={
+        <BitcoinConfirmationsSummary
+          minConfirmationsNeeded={requiredConfirmations}
+          txConfirmations={confirmations}
+        />
+      }
```

Comment:
I've also noticed that bitcoin confirmations are not updated when they happen on the chain 🤔 I'm not sure what's the issue yet though, so for now I just point that out. I had to refresh the deposit details page to display me the step 2.

Reply:
Hmm, that's true. Ain't got a clue why. I didn't change anything in this area. To prevent blocking this PR let's investigate in separate PR okay? The UI itself works fine.